README.md

```diff
@@ -78,6 +78,9 @@ Install node_modules used to build Sass files and run E2E tests
 ```
 cd web-languageforge
 npm install
+cd src
+bower install
```

Comment:
This shouldn't be needed. If you have just done the Ansible task in the instruction prior then it will have done a bower install.